### PR TITLE
23 thing model helpers

### LINKED ISSUE #23: Moving thing-model-helpers from node-wot
As agreed today, we can move https://github.com/eclipse-thingweb/node-wot/blob/master/packages/td-tools/src/thing-model-helpers.ts here.

This means creating the folder, CI pipeline, test. Then deleting from node-wot. Then publishing to npm with name `@thingweb/tm-helpers` and adjusting imports in playground.

@hasanheroglu will tackle this

### PR DESCRIPTION
**Pull request by hasanheroglu**
Closes #23

### CODE REVIEW

File: .github/workflows/upload-to-codecov.yaml
Comment (egekorkan) [inline .github/workflows/upload-to-codecov.yaml:1]: Shouldn't this be another PR?

File: node/tm-helpers/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Eclipse Thingweb - TM Helpers
+
+The package provides helper tools for Thing Models.
```
Comment (egekorkan) [inline node/tm-helpers/README.md:3]: explaining what can be done would be nice

File: node/tm-helpers/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "tm-helpers",
```
Comment (egekorkan) [inline node/tm-helpers/package.json:2]:
```suggestion
    "name": "@thingweb/tm-helpers",
```
but I feel we should find a nicer name. Any opinions @relu91 ?

**Review by egekorkan — CHANGES_REQUESTED** [review]

File: .github/workflows/upload-to-codecov.yaml
Comment (hasanheroglu) [reply .github/workflows/upload-to-codecov.yaml:1]: Yes, I removed this now.

File: node/tm-helpers/README.md (hunk 1 above)
Comment (hasanheroglu) [reply node/tm-helpers/README.md:3]: Updated [here](https://github.com/eclipse-thingweb/td-tools/pull/27/commits/1466ee625d181dc9703d74aec18cde44f98ff3fc).

File: node/tm-helpers/package.json (hunk 2 above)
Comment (relu91) [reply node/tm-helpers/package.json:2]: why don't we just call it `@thingweb/thing-model` ? It's clean but it might be mistaken as if we are offering a thing-model of @thingweb. I think we can solve this mistake with a simple description and readme.

Comment (egekorkan) [reply node/tm-helpers/package.json]: Now using this name :) The readme explains that it is helper tools to do some functions
